sections/04_Cryptography.md

```diff
@@ -0,0 +1,20 @@
+# Cryptography basics: Digital Signatures and Hash Functions
+
+Neo mainly uses SHA-256 and RIPEMD-160 functions for hashing.
+Digital Signatures are performed via elliptic curves (ECDSA), standard P-256 (secp256r1), which is quite similar to Bitcoin (secp256k1).
+
+## Current cryptography and NeoQS
```

Comment:
Remove

Reply:
The description about NeoQS is consistent with the White Paper and our state-of-the-art research about cryptographic system based on Lattice problems.

We believe that it should be kept.
Next year after NEO 3 we can keep describing it and, if necessary, change to another specification.

Reply:
Is there may be a chance to look more into EdDSA for NEO 3.0? It's more secure, as it doesn't rely on RNG for security, more resilient against side-channel attacks, has better performance, etc.

BLS is also hot right now, as it offers signature aggregation, which could be used to sign the whole block with just one signature (optimization) or to have native multisig wallets.

Reply:
The disadvantage of BLS is that the verification is too slow.

Reply:
I remember reading a good article by Dash about optimization of BLS: https://blog.dash.org/bls-is-it-really-that-slow-4ca8c1fcd38e

Reply:
Let's keep it as it is for now, it matches the White Paper with further info and is aligned with current perspectives.